certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+{
+  "x-generator": "NSwag v13.14.8.0 (NJsonSchema v10.6.1.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "AIForged",
+    "description": "Integration with AI Forged system",
+    "termsOfService": "None",
+    "contact": {
+      "name": "AIForged",
+      "url": "https://www.aiforged.com",
+      "email": "support@aiforged.com"
+    },
+    "license": {
+      "name": "Proprietary license"
```

Comment:
@aiforged Again, we need to get it validated by our licensing team which slows your certification process. I would recommend you to remove it.

Reply:
ok, will remove this